ProductClass.js

```diff
@@ -0,0 +1,26 @@
+export class PRODUCT {
+  constructor(name, description, price, tags, images) {
+    this.name = name;
+    this.description = description;
+    this.price = price;
+    this.tags = tags;
+    this.images = images;
+    this.찜꽁 = "💖x 0"; //찜꽁
```

Comment:
요구사항을 보면 favoriteCount라는 정확한 이름을 사용해야 합니다!
```suggestion
    this.favoriteCount = 0;
```